examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/components/navigation/.content.xml

```diff
@@ -3,5 +3,5 @@
     jcr:description="The top level navigation component for the CIF Components Library"
     jcr:primaryType="cq:Component"
     jcr:title="Top Navigation"
-    sling:resourceSuperType="core/cif/components/structure/navigation/v1/navigation"
```

Comment:
There should be an IT that tests the old component as well, since we still ship this.

Reply:
It should be covered now.